prometheus_api_client/prometheus_connect.py

```diff
@@ -350,25 +350,29 @@ def custom_query_range(self, query: str, start_time: datetime, end_time: datetim
             raise PrometheusApiClientException(
                 "HTTP Status Code {} ({})".format(response.status_code, response.content)
             )
-
         return data
 
     @retry(stop_max_attempt_number=MAX_REQUEST_RETRIES, wait_fixed=CONNECTION_RETRY_WAIT_TIME)
-    def get_metric_aggregation(self, query: str, operations: list, params: dict = None):
+    def get_metric_aggregation(self, query: str, operations: list, start_time: datetime,
+                               end_time: datetime, step: str, params: dict = None):
```

Comment:
hey @amitsagtani97, if we change from `custom_query` to `custom_query_range`, would the user still be able to run aggregates on current metric values?

Reply:
Would it be reasonable to make `start`, `end`, `step` args optional and then use `custom_query` or `custom_query_range` based on which args are passed? So that this aggregate functionality can be used regardless of whether a user is working on current metric values or values over time. WDYT? @4n4nd

Reply:
@chauhankaranraj @4n4nd Hey, I updated the pr and made start, end and step argument optional. Now it will support the aggregations on current metric values :)

Reply:
Thanks, @amitsagtani97!